README.md

```diff
@@ -30,6 +30,9 @@ OJP is part of the [Transmodel](https://www.transmodel-cen.eu/) family of standa
 * [OJP 1.0.1](https://github.com/VDVde/OJP/releases/tag/v1.0.1): Bugfix release
 * [OJP 1.0](https://github.com/VDVde/OJP/releases/tag/v1.0): The version described in CEN/TS 17118:2017
 
+## Documentation
+API documentation can be found [here](https://vdvde.github.io/OJP/).
+
 ## GitHub Repository
 Download the XSD schema files and examples in the git repository found at https://github.com/VDVde/OJP and take a look at the [documentation tables](https://vdvde.github.io/OJP/index.html).
```

Comment:
The link to the documentation tables is here as well. Do you want to keep it here or remove it?

Reply:
OK I will work there. and then we hope it works..